admin/src/scenes/etablissement/Create/Create.tsx

```diff
@@ -0,0 +1,172 @@
+        );
+        return setModalError(true);
+      }
+
+      if (e.code === ERRORS.ALREADY_EXISTS) {
+        setTitleError("Erreur : code UAI déjà utilisé !");
+        setTextError("Cet UAI est déjà utilisé sur la plateforme : la fiche de cet établissement a déjà été créée. Veuillez vérifier et recommencer svp.");
+        return setModalError(true);
+      }
+
+      if (e.code === ERRORS.NOT_FOUND) {
+        setTitleError("Erreur : UAI inconnu !");
+        setTextError("Cet UAI n'est pas reconnu dans l'annuaire des établissements. Veuillez vérifier et recommencer svp.");
+        return setModalError(true);
+      }
+
+      toastr.error("Oups, une erreur est survenue lors de la création de l'établissement", "");
+    }
+  };
+
+  return (
+    <Page>
+      <Header
+        title="Création d’un établissement"
+        breadcrumb={[{ title: <HiHome size={20} className="text-gray-400 hover:text-gray-500" />, to: "/" }, { title: "Séjours" }, { title: "Créer un établissement" }]}
+      />
+      <Container title="Informations générales">
+        <div className="flex items-stretch justify-between mt-8">
+          <div className="flex-1 shrink-0">
+            <div>
+              <Label title="UAI de l'établissement" name="uai" className="!font-[500]" />
+              <div className="flex items-center justify-between gap-3">
+                <InputText
+                  name="uai"
+                  className="flex-1"
+                  active={true}
+                  placeholder="Préciser"
+                  value={etablissement.uai!}
+                  error={errors.uai}
+                  max={8}
+                  onChange={(e) => setEtablissement({ ...etablissement, uai: e.target.value })}
+                />
+              </div>
+            </div>
+            <div className="mt-8">
+              <Label title="Adresse email de l'établissement" name="email" className="!font-[500]" />
+              <div className="flex items-center justify-between gap-3">
+                <InputText
+                  name="email"
+                  className="flex-1"
+                  active={true}
+                  placeholder="Préciser"
+                  value={etablissement.email!}
+                  error={errors.email}
+                  onChange={(e) => setEtablissement({ ...etablissement, email: e.target.value })}
+                />
+              </div>
+            </div>
+          </div>
+          <div className="mx-14 w-[1px] bg-gray-200 shrink-0">&nbsp;</div>
+          <div className="flex-1 shrink-0">
+            <div>
+              <Label title="Nom du chef d'établissement" name="lastName" className="!font-[500]" />
+              <div className="flex items-center justify-between gap-3">
+                <InputText
+                  name="lastName"
+                  className="flex-1"
+                  active={true}
```

Comment:
```suggestion
                  active
```
exactement même comportement, comme tu veux